src/test/java/org/openmrs/contrib/qaframework/test/EditDemographicTest.java

```diff
@@ -0,0 +1,62 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ * <p>
```

Comment:
do we need `<p>`? if not, cross check through all the files with header license and do the needful